antispam_bee.php

```diff
@@ -374,6 +380,8 @@ private static function _init_internal_vars() {
 		$salt        = defined( 'NONCE_SALT' ) ? NONCE_SALT : ABSPATH;
 		self::$_salt = substr( sha1( $salt ), 0, 10 );
 
+		self::$_secret_prefix = chr(rand(97,122));
```

Comment:
I think, this would mean, with every load the prefix could be different. So it could be `a` when the user submits the form and `b` after he submitted and ASB tries to determine the fields name

Reply:
strangley, it doesnt fail :joy: :thinking: